From 95fa70707f30d434c68e82d5feb27e6d82e028ea Mon Sep 17 00:00:00 2001
From: Ira W. Snyder <iws@ovro.caltech.edu>
Date: Mon, 7 Dec 2009 23:17:51 -0500
Subject: [PATCH 099/119] vbus-enet: fix sparse warnings

Fix the following sparse warnings:
drivers/net/vbus-enet.c:411:9: warning: Using plain integer as NULL pointer
drivers/net/vbus-enet.c:1250:24: warning: Using plain integer as NULL pointer
drivers/net/vbus-enet.c:1326:24: warning: Using plain integer as NULL pointer

Signed-off-by: Ira W. Snyder <iws@ovro.caltech.edu>
Signed-off-by: Gregory Haskins <ghaskins@novell.com>
---
 drivers/net/vbus-enet.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/vbus-enet.c b/drivers/net/vbus-enet.c
index 026e6d3..7f631c0 100644
--- a/drivers/net/vbus-enet.c
+++ b/drivers/net/vbus-enet.c
@@ -408,7 +408,7 @@ tx_setup(struct vbus_enet_priv *priv)
 		priv->pmtd.pool = pool;
 
 		ret = dev->ops->shm(dev, NULL, shmid, 0, pool, poollen,
-				    0, NULL, 0);
+				    NULL, NULL, 0);
 		BUG_ON(ret < 0);
 	}
 
@@ -1247,7 +1247,7 @@ vbus_enet_evq_negcap(struct vbus_enet_priv *priv, unsigned long count)
 		priv->evq.pool = pool;
 
 		ret = dev->ops->shm(dev, NULL, query.dpid, 0,
-				    pool, poollen, 0, NULL, 0);
+				    pool, poollen, NULL, NULL, 0);
 		if (ret < 0)
 			return ret;
 
@@ -1323,7 +1323,7 @@ vbus_enet_l4ro_negcap(struct vbus_enet_priv *priv, unsigned long count)
 		 * pre-mapped descriptor pool
 		 */
 		ret = dev->ops->shm(dev, NULL, query.dpid, 0,
-				    pool, poollen, 0, NULL, 0);
+				    pool, poollen, NULL, NULL, 0);
 		if (ret < 0) {
 			printk(KERN_ERR "Error registering L4RO pool: %d\n",
 			       ret);
-- 
1.6.5.2

